File: InGame/Packages/Baraja.js
import {Carta} from "./Carta.js";
export class Baraja{
    constructor(){
        this.baraja = [];
        let pintas = ["Clubs","Diamonds","Hearts","Spades"];
        for (let i = 0; i <= 3; i++){
            let pinta = pintas[i]
            for (let j = 2; j <= 14; j++){
                if (j == 11){
                    this.baraja.push(new Carta(pinta,"J"));
                }
                else if (j == 12){
                    this.baraja.push(new Carta(pinta,"Q"));
                }
                else if (j == 13){
                    this.baraja.push(new Carta(pinta,"K"));
                }
                else if (j == 14){
                    this.baraja.push(new Carta(pinta,"A"));
                }
                else{
                    this.baraja.push(new Carta(pinta,`${j}`));
                };
            };
        };
    };
};

File: Src/Packages/Baraja.py
from Src.Packages.Carta import Carta

class Baraja:
    def __init__(self):             ### Método para creación de una baraja ###
        self.__baraja = []
        Pintas = ["Clubs", "Diamonds","Hearts", "Spades"]
        for pinta in Pintas:
            for i in range(2,15):
                if i == 14:
                    self.__baraja.append(Carta(pinta, "A"))
                elif i == 13:
                    self.__baraja.append(Carta(pinta, "K"))
                elif i == 12:
                    self.__baraja.append(Carta(pinta,"Q"))
                elif i == 11:
                    self.__baraja.append(Carta(pinta, "J"))
                else:
                    self.__baraja.append(Carta(pinta, str(i)))
    
    def getBaraja(self):            ### Método para obtener la baraja ###
        return self.__baraja